publishers/sidekiq.go

```diff
@@ -95,5 +81,5 @@ func (lib *Sidekiq) scheduleJob(packageVersion data.PackageVersion) {
 		log.WithFields(log.Fields{"publisher": "sidekiq"}).Error(err)
 		return
 	}
-	lib.RedisClient.LPush(lib.Context, fmt.Sprintf("queue:%s", job.Queue), string(encoded))
+	redis.Client.LPush(lib.Context, fmt.Sprintf("queue:%s", job.Queue), string(encoded))
```

Comment:
I don't quite get how context.Background() works here. Do we want to worry about it from calling code or is it something we can do internal to the redis package?

Reply:
context.Background() is the initial context that we'd pass down into our callstack. If this were a web server, this would be the parent context that we'd derive child contexts from -- and if we cancel a context then its children are cancelled too.

I don't think contexts are as useful for Depper as they would be for a web server, since we don't need to cancel ingestors or publishers. But even if we did, IMO we should store a parent context in the Ingestor/Publisher instead of this sidekiq struct. 

In fact, the [context docs](https://golang.org/pkg/context/) say: `Do not store Contexts inside a struct type; instead, pass a Context explicitly to each function that needs it.`

Reply:
K, sounds like we should refactor this a little, but another time.

Reply:
BTW, GoTime did a good episode on context [here](https://changelog.com/gotime/143) for anyone who fancies podcasts.